core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -78,15 +102,6 @@ public boolean nullValue() {
     }
 
     /**
-     * Should the missing filter include documents where the field doesn't exist in the docs.
-     * Defaults to <tt>true</tt>.
-     */
-    public MissingQueryBuilder existence(boolean existence) {
```

Comment:
it all sounds good, but I still wonder why we didn't make those fields final. Can we do that? we only set them in the constructor no?